artemis/src/test/java/tech/pegasys/artemis/cli/deposit/CommonParamsTest.java

```diff
@@ -0,0 +1,138 @@
+    final File keystoreFile = tempDir.resolve(keystoreFileName).toFile();
+    // create password file
+    final File passwordFile =
+        Files.writeString(tempDir.resolve("password.txt"), "test123").toFile();
+
+    Eth1PrivateKeyOptions.Eth1EncryptedPrivateKeystoreOptions keystoreOptions =
+        new Eth1PrivateKeyOptions.Eth1EncryptedPrivateKeystoreOptions();
+    keystoreOptions.eth1PrivateKeystoreFile = keystoreFile;
+    keystoreOptions.eth1PrivateKeystorePasswordFile = passwordFile;
+
+    Eth1PrivateKeyOptions eth1PrivateKeyOptions = new Eth1PrivateKeyOptions();
+    eth1PrivateKeyOptions.keystoreOptions = keystoreOptions;
+
+    CommonParams commonParams = new CommonParams(commandSpec, eth1PrivateKeyOptions);
+    final Credentials eth1Credentials = commonParams.getEth1Credentials();
+    assertThat(eth1Credentials.getEcKeyPair()).isEqualTo(EXPECTED_EC_KEYPAIR);
+  }
+
+  @Test
+  void nonExistentEth1EncryptedKeystoreThrowsError(@TempDir final Path tempDir)
+      throws IOException, CipherException {
+    Eth1PrivateKeyOptions.Eth1EncryptedPrivateKeystoreOptions keystoreOptions =
+        new Eth1PrivateKeyOptions.Eth1EncryptedPrivateKeystoreOptions();
+    keystoreOptions.eth1PrivateKeystoreFile = tempDir.resolve("nonExistent").toFile();
+    keystoreOptions.eth1PrivateKeystorePasswordFile = tempDir.resolve("nonExistent").toFile();
+
+    Eth1PrivateKeyOptions eth1PrivateKeyOptions = new Eth1PrivateKeyOptions();
+    eth1PrivateKeyOptions.keystoreOptions = keystoreOptions;
+
+    CommonParams commonParams = new CommonParams(commandSpec, eth1PrivateKeyOptions);
+
+    when(commandSpec.commandLine()).thenReturn(mock(CommandLine.class));
+    assertThatExceptionOfType(CommandLine.ParameterException.class)
+        .isThrownBy(() -> commonParams.getEth1Credentials())
+        .withMessage("Error: File not found: " + keystoreOptions.eth1PrivateKeystoreFile);
+  }
+
+  @Test
+  void invalidEth1EncryptedKeystoreThrowsError(@TempDir final Path tempDir) throws IOException {
+    Eth1PrivateKeyOptions.Eth1EncryptedPrivateKeystoreOptions keystoreOptions =
+        new Eth1PrivateKeyOptions.Eth1EncryptedPrivateKeystoreOptions();
+    keystoreOptions.eth1PrivateKeystoreFile =
+        Files.writeString(tempDir.resolve("v3.json"), "{test:123}").toFile();
+    keystoreOptions.eth1PrivateKeystorePasswordFile =
+        Files.writeString(tempDir.resolve("password.txt"), "test123").toFile();
+
+    Eth1PrivateKeyOptions eth1PrivateKeyOptions = new Eth1PrivateKeyOptions();
+    eth1PrivateKeyOptions.keystoreOptions = keystoreOptions;
+
+    CommonParams commonParams = new CommonParams(commandSpec, eth1PrivateKeyOptions);
+
+    when(commandSpec.commandLine()).thenReturn(mock(CommandLine.class));
+    assertThatExceptionOfType(CommandLine.ParameterException.class)
+        .isThrownBy(() -> commonParams.getEth1Credentials())
+        .withMessage(
+            "Error: Unable to decrypt v3 keystore [%s] : Wallet version is not supported",
```

Comment:
From this error, we testing that the incorrect version of the wallet store was provided? Can the test be renamed to make this clearer?

Reply:
The intent was to check a valid json which is not compliant with keystore format. I'll rename the method accordingly.